### DISCUSSION TITLE
Predefined System Metadata “title”, “alt” and “copyright”

### DISCUSSION THREAD

**Discussion by hethehe** [pimcore/pimcore#17858] [Q&A / Support]
See: https://pimcore.com/docs/platform/Pimcore/Assets/Working_with_PHP_API/#predefined-system-metadata

At least since Pimcore 11.3.x, “title”, “alt” and “copyright” are no longer generated automatically when I upload an image.

Is this intentional or a bug?

**Comment by kingjia90** [comment]
Thank you for reporting!

Might be a bug, a regression from https://github.com/pimcore/pimcore/pull/16861 ?

**Comment by hethehe** [comment]
I believe https://github.com/pimcore/admin-ui-classic-bundle/pull/524 was the change.

`title`, `alt` and `copyright` are displayed by default if no custom metadata is set (only for images). For example, if `title` is defined, `alt` and `copyright` are no longer displayed.

It looks like the above change has somehow overridden the default output.

**Reply by SaschaHowe** [reply]
> I believe [pimcore/admin-ui-classic-bundle#524](https://github.com/pimcore/admin-ui-classic-bundle/pull/524) was the change.
> 
> `title`, `alt` and `copyright` are displayed by default if no custom metadata is set (only for images). For example, if `title` is defined, `alt` and `copyright` are no longer displayed.
> 
> It looks like the above change has somehow overridden the default output.

I tried rolling back these changes locally and for me it still didnt' work after that.

**Comment by kingjia90** [comment]
It would be nice, if you could put some breakpoints on these lines and before any condition in the IF block to see if it's reachable
![image](https://github.com/user-attachments/assets/f1a170eb-43c8-466d-879a-83578c2ff9ac)

So far i haven't reproduced it yet, but it's not intentional

**Reply by SaschaHowe** [reply]
Hey, 

in my test these lines were not reached. 
is there a proper GitHub issue for this or only this discussion?

**Reply by kingjia90** [reply]
no afaik, please open an issue as bug if it's still occurring, and we will take a look there

Thank you in advance!
